src/components/Tombstone.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <dl id="tombstone">
+    <!-- Registration Date -->
+    <template v-if="!!registrationDate">
+      <dt class="mr-2">Registration Date:</dt>
+      <dd id="registration-date">{{ registrationDate }}</dd>
+    </template>
+
+    <!-- Registration Number -->
+    <template v-if="!!registrationNumber">
+      <dt class="mr-2">Registration Number:</dt>
+      <dd id="registration-number">{{ registrationNumber }}</dd>
+    </template>
+
+    <!-- Business Number -->
+    <template v-if="!!businessNumber">
+      <dt class="mr-2">Business Number:</dt>
+      <dd id="business-number">{{ businessNumber }}</dd>
+    </template>
+
+    <!-- Incorporation Number -->
+    <template v-if="!!incorporationNumber">
+      <dt class="mr-2">Incorporation Number:</dt>
+      <dd id="incorporation-number">{{ incorporationNumber }}</dd>
+    </template>
+
+    <!-- NR Number -->
+    <template v-if="!!nameRequestNumber">
+      <dt class="mr-2">Name Request Number:</dt>
+      <dd id="name-request-number">{{ nameRequestNumber }}</dd>
+    </template>
+
+    <!-- Email -->
+    <template v-if="!!email">
+      <dt class="mr-2">Email:</dt>
+      <dd id="email" class="cursor-pointer" @click="editBusinessProfile()">
+        <span>{{ email }}</span>
+        <v-btn id="change-email-button" small text color="primary">
+          <v-icon small>mdi-pencil</v-icon>
+          <span>Change</span>
+        </v-btn>
+      </dd>
+    </template>
+
+    <!-- Phone -->
+    <template v-if="!!phone">
+      <dt class="mr-2">Phone:</dt>
+      <dd id="phone" class="cursor-pointer" @click="editBusinessProfile()">
+        <span>{{ phone }}</span>
+        <v-btn id="change-phone-button" small text color="primary">
+          <v-icon small>mdi-pencil</v-icon>
+          <span>Change</span>
+        </v-btn>
+      </dd>
+    </template>
+  </dl>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { navigate } from '@/utils'
+import { DateUtilities } from '@/services'
+
+@Component({})
+export default class Tombstone extends Vue {
```

Comment:
What about using the Options API so it's compatible with Vue3?

Reply:
See other comment. I want to move forward this way for now. This shouldn't be much additional work to change later if we change all other components over at the same time.

Reply:
You're correct, the Class API is supported in Vue3.  Please ignore the suggestion.

Reply:
Are you OK to resolve this conversation?